packages/@aws-cdk/aws-backup/lib/vault.ts

```diff
@@ -223,9 +214,31 @@ export class BackupVault extends BackupVaultBase {
       props.notificationTopic.grantPublish(new iam.ServicePrincipal('backup.amazonaws.com'));
     }
 
+    if (props.blockRecoveryPointDeletion && props.accessPolicy) {
+      throw new Error('Cannot specify `accessPolicy` when `blockRecoveryPointDeletion` is set to `true`');
+    }
```

Comment:
is there a particular reason you went this route instead of just combining the two policies?

Reply:
I thought it's either you fully specify your own policy or you use a prop that configures it for you. We can merge the IAM statements together if you think it's more intuitive. Let me know.

Reply:
Yes, I believe the statements should be merged as it would reduce the usability of this feature if custom access policies could not be provided at the same time